llvm/test/tools/sycl-post-link/device-code-split/per-aspect-split-1.ll

```diff
@@ -10,29 +10,29 @@
 ; RUN: sycl-post-link -split=auto -symbols -S < %s -o %t.table
 ; RUN: FileCheck %s -input-file=%t_0.ll --check-prefixes CHECK-M0-IR \
 ; RUN: --implicit-check-not kernel0 --implicit-check-not kernel1
-; RUN: FileCheck %s -input-file=%t_1.ll --check-prefixes CHECK-M1-IR \
+; RUN: FileCheck %s -input-file=%t_2.ll --check-prefixes CHECK-M1-IR \
```

Comment:
These test updates were triggered by the fact that the order of the entries in the .table file changed due to the addition of the new entry in the list optional kernel features. A better way to NOT depend on the order of the entries is required in the long run.